code/game/objects/items/bodybag.dm

```diff
@@ -1,33 +1,31 @@
 //Also contains /obj/structure/closet/body_bag because I doubt anyone would think to look for bodybags in /object/structures
-
+#define STASIS_DELAY 6000
+//ten minutes
 /obj/item/bodybag
 	name = "body bag"
 	desc = "A folded bag designed for the storage and transportation of cadavers."
 	icon = 'icons/obj/bodybag.dmi'
 	icon_state = "bodybag_folded"
 	w_class = WEIGHT_CLASS_SMALL
 
-	attack_self(mob/user)
-		var/obj/structure/closet/body_bag/R = new /obj/structure/closet/body_bag(user.loc)
-		R.add_fingerprint(user)
-		qdel(src)
+/obj/item/bodybag/attack_self(mob/user)
+	var/obj/structure/closet/body_bag/R = new /obj/structure/closet/body_bag(user.loc)

```

Comment:
Please address this- As it is, this allows for opening bodybags inside places such as spacepods or mecha where they will become completely unrecoverable. Either check to make sure the user's location is a turf, or change this to a get_turf.